src/types.ts

```diff
@@ -22,8 +22,13 @@ export type GetKeysOptions = TransactionOptions & {
   maxDepth?: number;
 };
 
+export interface DriverFlags {
+  supportsMaxDepth?: boolean;
```

Comment:
I assume (most of the) options will be "capabilities" which imply `supports` part. It can reduce verbosity and few bytes in bundles to omit.

```suggestion
  maxDepth?: boolean;
  ttl?: boolean;
```